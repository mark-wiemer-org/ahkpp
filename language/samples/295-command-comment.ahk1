; [Issue #295](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/295)
ToolTip ;Comment
ToolTip;NotComment
