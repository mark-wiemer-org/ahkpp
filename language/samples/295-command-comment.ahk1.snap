>; [Issue #295](https://github.com/mark-wiemer-org/ahkpp/issues/295)
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>ToolTip ;Comment
#^^^^^^^ source.ahk support.function.ahk
#       ^ source.ahk comment.line.semicolon.ahk
#        ^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
#         ^^^^^^^ source.ahk comment.line.semicolon.ahk
>ToolTip;NotComment
#^^^^^^^ source.ahk support.function.ahk
#       ^ source.ahk
#        ^^^^^^^^^^ source.ahk variable.def.ahk
>